robotd/devices.py

```diff
@@ -97,6 +97,30 @@ def status(self):
         return {'temperature': temp_milli_degrees / 1000}
 
 
+class GameState(Board):
+    """
+    State storage for the game. keeps a store of whet it is sent
+    """
+
+    # define the name od the board
+    board_type_id = 'game'
+    create_on_startup = True
+
+    def __init__(self):
+        super().__init__({})
+        self._state = {'corner': 0, 'mode': 'development'}
```

Comment:
And should we be calling it 'corner' rather than, say, 'zone'? It does mean changing the API if robots start not in corners in a future game.